content/documentation.rst

```diff
@@ -242,6 +389,12 @@ For example, instead of 'user', add a real name such as 'alice' or 'bob':
     home_folder_path = /PATH/TO/MARK/HOME
     password = PASSWORD
 
+Ensure to add examples for the text file configuration as first priority,
+followed by steps in the Local Manager GUI.
+
+The configuration / log examples are added *after* a narrative description of
```

Comment:
I think at some point we will need to break the text configuration/Local Manager UI configuration into separate chapters.